client/src/modules/roles/roles.ctrl.js

```diff
@@ -92,6 +92,7 @@ function RolesController($uibModal, Roles, Session, Modal, Notify, bhConstants)
     headerCellFilter : 'translate',
   }, {
     field : 'actions',
+    enableFiltering : false,
```

Comment:
Glad you figured this out!  I suppose I should have mentioned it 😄

Reply:
Haha, no worries! It's good for me to learn through searching, trial and error, and such. 👍